e2e/cypress/integration/messaging/visual_verification_of_tooltips_on_top_nav_channel_icons_posts_spec.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    let testTeam;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Set up test channel
+            cy.apiCreateChannel(testTeam.id, 'channel-test', 'Public channel with a long name');
+
+            // # Set up Demo plugin
+            cy.apiInstallPluginFromUrl('https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.8.0/com.mattermost.demo-plugin-0.8.0.tar.gz', true);
+            cy.visit('/admin_console/plugins/plugin_com.mattermost.demo-plugin');
+            cy.get('[data-testid="PluginSettings.PluginStates.com+mattermost+demo-plugin.Enabletrue"]').check().should('be', 'checked');
```

Comment:
Correct way to get by test Id is this:
`cy.findByTestId('PluginSettings.PluginStates.com+mattermost+demo-plugin.Enabletrue');`

Also, as mentioned earlier, please use the following pattern to verify if its visible or if its checked:
`.should('be.checked');`

Reply:
This has been fixed by using the apiEnablePlugin - I'll be pushing an update later this morning.